js/search.js

```diff
@@ -73,7 +77,7 @@
 			}
 
 			this.activeSearch = searchText;
-			this.$element.addClass('searched');
+			this.$element.addClass('searched pending');
```

Comment:
Thats actually how I wrote it when we had our 1:1. If you'll remember, I had a mini-rant about storing state in classnames, but decided to follow convention. If I can't re-write it all, I honestly don't care either way, so if anyone has a preference, I can follow popular opinion.